modules/libcom/src/misc/aToIPAddr.c

```diff
@@ -90,6 +90,13 @@ aToIPAddr( const char *pAddrString, unsigned short defaultPort,
     unsigned port;
     struct in_addr ina;
 
+    /*
+     * Check if input string is null
+     */
+    if ( pAddrString == NULL ) {
```

Comment:
While it is in keeping with the existing style in this function, I think it can be assumed that a reader will understand `== NULL` to be a null check.  So the comment can be omitted.